docs/blog/newsletter-september-2022.rst

```diff
@@ -0,0 +1,91 @@
+.. post:: September 06, 2022
+   :tags: newsletter
+
+##########################################
+Write the Docs Newsletter – September 2022
+##########################################
+
+Hey, everyone, the newsletter is back! We took one of our regularly scheduled breaks, but now we're back with regularly exceptional insights from our community.
+
+If you'd like to get more of those insights, it's your last chance to get `tickets for the Prague conference <https://www.writethedocs.org/conf/prague/2022/tickets/>`__. Ticket sales end on September 8. Then the conference starts September 11. We'll certainly all have lots of new ideas to think about afterwards.
```

Comment:
"More of these insights" confused me intially. how about "If you'd like to get more insight s from the WTD community, now is your last...."